pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -49,6 +49,9 @@ var (
 func NewCmdRolloutHistory(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &HistoryOptions{}
 
+	validArgs := []string{"deployment"}
```

Comment:
We already have ported `rollout history`. Make sure this is configurable because it needs to work with deploymentconfigs too